### ISSUE TITLE
Button without text label in message box.

### ISSUE THREAD

**Issue by ikbenkous** [ea4k/klog#641]
Will come back later to add more text info regarding this. I clicked "add" trying to log a fake QSO for debugging purposes.
And then this happened:
![image](https://github.com/ea4k/klog/assets/87572049/1e9ffafa-2ad9-4cf7-b5b3-dfaf2548f4df)

Maybe the buttun should say `leave entity empty` or something?

**Comment by ikbenkous** [comment]
Maybe rephrase this dialog message to be BLUF (Bottom Line Up Front)?

I'll be going over this and other text in klog semi-soonish.

**Comment by ea4k** [comment]
I can't reproduce... can you give me the steps? I think I have eventually seen that but I can't reproduce
